libp2p/examples/kademlia.rs

```diff
@@ -155,12 +156,12 @@ fn main() {
 }
 
 /// Stores initial addresses on the given peer store. Uses a very large timeout.
-pub fn ipfs_bootstrap<P>(peer_store: P)
+pub fn ipfs_bootstrap<P>(peer_interface: P)
 where
     P: Peerstore + Clone,
 {
     const ADDRESSES: &[&str] = &[
-        "/ip4/127.0.0.1/tcp/4001/ipfs/QmaCpDMGvV2BGHeYERUEnRQAwe3N8SzbUtfsmvsqQLuvuJ",
+        "/ip4/104.131.131.82/tcp/4001/ipfs/QmaCpDMGvV2BGHeYERUEnRQAwe3N8SzbUtfsmvsqQLuvuJ",
```

Comment:
Out of curiosity, where does this IP come from?

Reply:
Oh, that's an IPFS bootstrap node. It's an accident.